sinks/cloudwatch/cloudwatch_test.go

```diff
@@ -102,8 +103,16 @@ func TestFlush(t *testing.T) {
 	}(reqBodyCh, done)
 
 	// Flush the sink
-	_, err = sink.Flush(context.Background(), metrics)
+	flushResult, err := sink.Flush(context.Background(), metrics)
 	assert.NoError(t, err)
+	assert.Equal(t,
+		sinks.MetricFlushResult{
+			MetricsFlushed: 3,
+			MetricsSkipped: 0,
```

Comment:
You can omit L111 and L112 - https://stackoverflow.com/questions/28625794/structs-zero-value